### PR TITLE
Private/gokay/latlng

### PR DESCRIPTION
**Pull request by gokaysatir**
* Resolves: # 
* Target version: master 

### TODO

- [ ] ...

### Checklist

- [ ] Code is properly formatted
- [ ] All commits have Change-Id
- [ ] I have run tests with `make check`
- [ ] I have issued `make run` and manually verified that everything looks okay
- [ ] Documentation (manuals or wiki) has been updated or is not required

### CODE REVIEW

File: browser/src/layer/tile/CanvasTileLayer.js [hunk 1]
```diff
@@ -4457,7 +4368,6 @@ L.CanvasTileLayer = L.Layer.extend({
 		if (this._docType === 'spreadsheet') {
 			map.on('zoomend', this._onCellCursorShift, this);
 		}
-		map.on('zoomend', L.bind(this.eachView, this, this._viewCursors, this._onUpdateViewCursor, this, false));
```
Comment (eszkadev) [inline browser/src/layer/tile/CanvasTileLayer.js:4466]: this handler is removed, but do we have some replacement which happens after zoom change?

Comment (gokaysatir) [reply browser/src/layer/tile/CanvasTileLayer.js:4466]: > this handler is removed, but do we have some replacement which happens after zoom change?

Thanks for checking. Yes we a call under "_onZoomEnd" function.

**Comment by eszkadev** [comment]
This PR works nice, but I found bug with user label. When using different sheets, we see label from the user who isn't on our sheet.

[latlng-2024-07-04_10.12.20.webm](https://github.com/CollaboraOnline/online/assets/5307369/5cccce57-6d2f-4537-9f7f-ce86b65d948e)

**Review by eszkadev — APPROVED** [review]
looks ok

**Comment by gokaysatir** [comment]
> This PR works nice, but I found bug with user label. When using different sheets, we see label from the user who isn't on our sheet.
> 
>  latlng-2024-07-04_10.12.20.webm

Thanks for the catch :) Fixed.

**Pull request merged**